systemtest/src/main/java/io/strimzi/systemtest/utils/VerificationUtils.java

```diff
@@ -0,0 +1,463 @@
+import io.strimzi.systemtest.Environment;
+import io.strimzi.systemtest.resources.crd.KafkaMirrorMaker2Resource;
+import io.strimzi.systemtest.resources.crd.KafkaResource;
+import io.strimzi.systemtest.utils.kubeUtils.objects.PodUtils;
+import io.strimzi.test.TestUtils;
+import io.strimzi.test.k8s.KubeClusterResource;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import static io.strimzi.systemtest.utils.StUtils.getImagesFromConfig;
+import static io.strimzi.test.k8s.KubeClusterResource.cmdKubeClient;
+import static io.strimzi.test.k8s.KubeClusterResource.kubeClient;
+import static java.util.Arrays.asList;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.IsNull.nullValue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+/**
+ * Utility class for verification and validation tasks related to resource requests, labels, environment variables, and component configurations.
+ * This class provides static methods for performing checks commonly used for containers.
+ */
+public class VerificationUtils {
+
+    private static final Logger LOGGER = LogManager.getLogger(VerificationUtils.class);
+
+    private static final String KAFKA_IMAGE_MAP = "STRIMZI_KAFKA_IMAGES";
+    private static final String TO_IMAGE = "STRIMZI_DEFAULT_TOPIC_OPERATOR_IMAGE";
+    private static final String UO_IMAGE = "STRIMZI_DEFAULT_USER_OPERATOR_IMAGE";
+    private static final String KAFKA_INIT_IMAGE = "STRIMZI_DEFAULT_KAFKA_INIT_IMAGE";
+    private static final String TLS_SIDECAR_EO_IMAGE = "STRIMZI_DEFAULT_TLS_SIDECAR_ENTITY_OPERATOR_IMAGE";
+    private static final String KAFKA_MIRROR_MAKER_2_IMAGE_MAP = "STRIMZI_KAFKA_MIRROR_MAKER_2_IMAGES";
+    private static final String KAFKA_CONNECT_IMAGE_MAP = "STRIMZI_KAFKA_CONNECT_IMAGES";
+
+    /**
+     * Verifies expected and current resources requests inside selected pods container
+     * @param namespace Namespace name where is container is located
+     * @param podName Name of pod where container is located
+     * @param containerName The container where verifying is expected
+     * @param memoryLimit expected value for allowed memory resource limitation
+     * @param cpuLimit expected value for allowed CPU resource limitation
+     * @param memoryRequest expected value for requested memory size
+     * @param cpuRequest expected value for requested CPU in CPU units
+     */
+    public static void assertPodResourceRequests(String namespace, String podName, String containerName, String memoryLimit, String cpuLimit, String memoryRequest, String cpuRequest) {
+        Pod po = kubeClient(namespace).getPod(namespace, podName);
```

Comment:
you don't need to specify namespace in `kubeClient()` right?

Reply:
changed it for all instances